audits/sql/utils/to_pg_utils.go

```diff
@@ -0,0 +1,75 @@
+package utils
```

Comment:
`[Suggestion]` not a big fan of naming this package `utils`, since it may lead to conflicts with other packages in the future and it's quite ambiguous for other developers. 

I'd suggest going with something more specific like 


```suggestion
package pgconvert
```

Reply:
resolved! Thanks.